uxarray/core/api.py

```diff
@@ -61,6 +61,8 @@ def open_grid(
     >>> uxgrid = ux.open_grid("grid_filename.g")
     """
 
+    # extension = os.path.splitext(grid_filename_or_obj)
+
     if "source_grid" in kwargs.keys():
```

Comment:
We can extend the API to support the different geometry files using `open_grid` and `open_dataset` in a separate PR, unless you'd like to do so in this one.

Reply:
how about a new PR for this and the notebook that you mention in the next comment?